src/Actions/CollectFrontendTranslations.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Rapidez\Core\Actions;
+
+use Illuminate\Support\Facades\App;
+use Illuminate\Support\Facades\File;
+use Illuminate\Translation\Translator;
+
+class CollectFrontendTranslations
+{
+    public static function collect(): array
+    {
+        /** @var Translator $translator */
+        $translator = app(Translator::class);
+        $loader = $translator->getLoader();
+
+        $allTranslations = collect($loader->namespaces())
+            ->filter(function (string $path, string $namespace): bool {
+                return str($namespace)->startsWith('rapidez');
```

Comment:
Is this not too restrictive? This would only allow official Rapidez packages to add frontend translations, right? What if a different vendor wants to make a package for Rapidez that includes frontend translations?